SourceX/miniwin/misc.cpp

```diff
@@ -167,7 +167,11 @@ bool SpawnWindow(LPCSTR lpWindowName, int nWidth, int nHeight)
 		flags |= SDL_WINDOW_INPUT_GRABBED;
 	}
 
+#ifndef SWITCH
 	window = SDL_CreateWindow(lpWindowName, SDL_WINDOWPOS_UNDEFINED, SDL_WINDOWPOS_UNDEFINED, nWidth, nHeight, flags);
+#else
+	window = SDL_CreateWindow(lpWindowName, 0, 0, 1920, 1080, 0);

```

Comment:
Is SDL unable to query the resolution size on the Switch?

Reply:
Indeed it is unable to.